### PR TITLE
Supports no parameters by Deprecated

### PR DESCRIPTION
**Pull request by momomomo111**
The existing implementation was such that if the @FlagType parameters (`owner`, `description`, `expiryDate`) were not set to values, the build would fail due to build errors.
We have provided DeprecatedInfo for version compatibility of this library.
By setting the value of DeprecatedInfo to the @FlagType parameter, you can temporarily avoid build errors

### CODE REVIEW

File: .github/workflows/lintIssues.yml [hunk 1]
```diff
@@ -28,4 +28,4 @@ jobs:
           HEAD_SHA: ${{ github.sha }}
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
-          kotlin scripts/create-issues-from-sarif.main.kts
+          kotlin scripts/maintain-flagfit-expiration-issue.main.kts
```
Comment (momomomo111) [inline .github/workflows/lintIssues.yml:31]: This is a point you pointed out in a previous PR

File: flagfit-flagtype/src/main/java/tv/abema/flagfit/DeprecatedInfo.kt [hunk 2]
```diff
@@ -0,0 +1,12 @@
+package tv.abema.flagfit
+
+object DeprecatedInfo {
```
Comment (takahirom) [inline flagfit-flagtype/src/main/java/tv/abema/flagfit/DeprecatedInfo.kt:3]: How about using this? I think `info` and `unknown` don't have meaning. 👀 

```kotlin
object FlagfitDeprecatedParams {
  @Deprecated("Flag with no assigned owner")
  const val OWNER_NOT_DEFINED = "OWNER_NOT_DEFINED"

  @Deprecated("Flag without a description")
  const val DESCRIPTION_NOT_DEFINED = "DESCRIPTION_NOT_DEFINED"

  @Deprecated("Flag without an expiry date")
  const val EXPIRY_DATE_NOT_DEFINED = "EXPIRY_DATE_NOT_DEFINED"
}
```

File: .github/workflows/lintIssues.yml (hunk 1 above)
Comment (takahirom) [reply .github/workflows/lintIssues.yml:31]: Just a friendly reminder that I had previously added another comment. https://github.com/abema/flagfit/pull/12#discussion_r1228804179

**Review by takahirom — APPROVED** [review]
I've commented some points. Please check. 🙏  But It's almost LGTM 👍

File: flagfit-flagtype/src/main/java/tv/abema/flagfit/DeprecatedInfo.kt (hunk 2 above)
Comment (momomomo111) [reply flagfit-flagtype/src/main/java/tv/abema/flagfit/DeprecatedInfo.kt]: Surely your suggestion is clearer and more understandable in its meaning. thank you.

b9094b1a906c362c505d98fbb2f2002ad6d4c165

**Pull request merged**